css/properties/justify-content.json

```diff
@@ -12,16 +12,25 @@
             "chrome": [
               {
                 "version_added": "29",
-                "notes": "Older versions of the specification treat absolutely positioned children as though they are 0 by 0 flex items. Later versions of the specification take them out of the flow and set their positions based on align and justify properties. Versions 52 and later implement the new behavior."
+                "partial_implementation": true,
+                "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
```

Comment:
@jpmedley Can you confirm this?

Reply:
It is correct. https://www.chromestatus.com/feature/6600926009753600